ci-operator/step-registry/rosa/sts/cluster/provision/rosa-sts-cluster-provision-ref.yaml

```diff
@@ -84,6 +84,9 @@ ref:
   - name: CLUSTER_TIMEOUT
     default: "7200"
     documentation: Set to number of seconds for the cluster to timeout if it's not ready.
-
+  - name: ENABLE_BYOVPC
+    default: "false"
+    documentation: Set to true to instruct the rosa cli to use a pre-existing VPC. When set to true the subnets belonging to the vpc will be provided on the rosa command line. The vpc must already exist prior to creating the cluster. 
```

Comment:
I do not think it is reasonable that whether using the VPC/Subnet_ids, it is not based on whether the user set `ENABLE_BYOVPC=true`, it is based on whether the cluster is a HCP cluster or the cluster is a proxy cluster, or the cluster is a private cluster, and so on.  To use the VPC should be judged in the code internal.

Reply:
I guess I misunderstood, I though the purpose of ENABLE_BYOVPC would be for cases where we want to test rosa clusters with byovpc where it is optional, such as a classic sts cluster with vpc subnets provided at creation time.

Reply:
So this variable is not available any more.